web-common/src/features/custom-dashboards/CustomDashboardPreview.svelte

```diff
@@ -164,65 +163,37 @@
 
 <svelte:window on:mouseup={handleMouseUp} on:mousemove={handleMouseMove} />
 
-<div bind:contentRect class="wrapper">
-  {#if showGrid || changing}
-    <GridLines {gridCell} {scrollOffset} {gapSize} {radius} {scale} />
-  {/if}
-  <div
-    role="presentation"
-    class="size-full overflow-y-auto overflow-x-hidden relative"
-    on:scroll={handleScroll}
-    on:click|self={deselect}
-  >
-    <div
-      class="dash pointer-events-none"
-      role="presentation"
-      style:width="{DEFAULT_WIDTH}px"
-      style:height="{maxBottom * gridCell}px"
-      style:transform="scale({scale})"
-    >
-      {#each charts as chart, i (i)}
-        {@const selected = i === selectedIndex}
-        {@const interacting = selected && changing}
-        {#if chart.chart}
-          <Element
-            {scale}
-            {i}
-            {chart}
-            {radius}
-            {selected}
-            {interacting}
-            {gapSize}
-            width={interacting
-              ? finalResize[0]
-              : Number(chart.width) * gridCell}
-            height={interacting
-              ? finalResize[1]
-              : Number(chart.height) * gridCell}
-            top={interacting ? finalDrag[1] : Number(chart.y) * gridCell}
-            left={interacting ? finalDrag[0] : Number(chart.x) * gridCell}
-            on:change={handleChange}
-          />
-        {/if}
-      {/each}
-    </div>
-  </div>
-</div>
-
-<style lang="postcss">
-  .wrapper {
-    width: 100%;
-    height: 100%;
-    position: relative;
-    overflow: hidden;
-    user-select: none;
-    margin: 0;
-    pointer-events: auto;
-  }
-
-  .dash {
-    transform-origin: top left;
-    position: absolute;
-    touch-action: none;
-  }
-</style>
+<Wrapper
+  width={DEFAULT_WIDTH}
+  height={maxBottom * gridCell}
+  {scale}
+  {showGrid}
+  {gapSize}
+  {gridCell}
+  {radius}
+  {changing}
+  bind:contentRect
+  on:scroll={handleScroll}
+  on:click={deselect}
+>
+  {#each charts as chart, i (i)}
```

Comment:
NIT: Should use a more generic wording then charts as we will also render non-charts